gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -1071,13 +1071,13 @@ public class ConfigurationKeys {
    * Configuration properties related to Flows
    */
   public static final String FLOW_RUN_IMMEDIATELY = "flow.runImmediately";
-  public static final String GOBBLIN_FLOW_SLA_TIME = "gobblin.flow.sla.time";
-  public static final String GOBBLIN_FLOW_SLA_TIME_UNIT = "gobblin.flow.sla.timeunit";
-  public static final String DEFAULT_GOBBLIN_FLOW_SLA_TIME_UNIT = TimeUnit.MINUTES.name();
-  public static final String GOBBLIN_JOB_START_SLA_TIME = "gobblin.job.start.sla.time";
-  public static final String GOBBLIN_JOB_START_SLA_TIME_UNIT = "gobblin.job.start.sla.timeunit";
-  public static final long FALLBACK_GOBBLIN_JOB_START_SLA_TIME = 10L;
-  public static final String FALLBACK_GOBBLIN_JOB_START_SLA_TIME_UNIT = TimeUnit.MINUTES.name();
+  public static final String GOBBLIN_FLOW_DEADLINE_TIME = "gobblin.flow.deadline.time";
+  public static final String GOBBLIN_FLOW_DEADLINE_TIME_UNIT = "gobblin.flow.deadline.timeunit";
```

Comment:
suggest `gobblin.flow.completion.deadline.time` or `gobblin.flow.finish.deadline.time` to parallel `gobblin.job.start.deadline.time` (rather than dropping one segment entirely)